### PR TITLE
Added virTemplate, supporting data, and references 

### PR DESCRIPTION
**Pull request by MAM122**
I completed the virTemplate and added all my A3 (and Tutorial 2 short story) into the template. I also added my supporting images and my references.bib (references file for Rmarkdown).

### CODE REVIEW

File: Images/Alignment.png
Comment (ababaian) [inline Images/Alignment.png:1]: This file is 0 bytes and not uploaded. Also for all images please include them in `viruses/img/<virus_name>/` folder as indiciated in the instructions.

File: virTemplate.md
Comment (ababaian) [inline virTemplate.md:1]: Please rename file after the virus, not the template file. For the template file, do not change it in the PR.

File: virTemplate.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+## Results
+
+### Ecology subsection A...
```
Comment (ababaian) [inline virTemplate.md]: Give your markdown meaningful sub-titles as you're presenting data, not submitting homework here.

File: virTemplate.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+For the SRR13764151 (Ebo) and SRR13764152 (Em) dataset, the analysis tab of the metadata shows the taxonomical species in the sample. However, unfortunately for the SRR13764153 (Eff) dataset, it does not allow me to see the taxonomy. For both SRR13764151 and SRR13764152 datasets, I see Plasmodium was detected (the RNA sequence of Plasmodium was detected). This parasite is transferred from mosquitoes to humans [@churcher2017probability]. In addition, discistrovirdiae are known to only infect insects [@steward2013marine]. In a study by Fauver et al., dicistrovirus was found in the Anopheles species of mosquitoes [@fauver2016west]. Female Anopheles mosquitoes are known to be hosts of Plasmodium [@mota2019parasitism]. Therefore, it is possible that the humans were bit by Anopheles mosquitoes who then transferred a dicistrovirus into them. Although Plasmodium is also in SRR13764153 (Ebo) but dicistrovirus is not in those cells, the Ebo cells may have a phenotype that prevents the dicistrovirus from being able to infect or replicate in that cell.
+
+### Virus Genome {Q3}
```
Comment (ababaian) [inline virTemplate.md]: Remove Q3 from sub-title, and worth changing "Virus Genome" to "Dicistrovirus Genome Fragment" which is better at communicating this is not a complete genome.

File: virTemplate.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+![](Images/Genome_map.png)
```
Comment (ababaian) [inline virTemplate.md]: Please store your images in `viruses/img/<virus_name>/` folder structure as indiciated in the instructions.

File: virTemplate.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+**Figure 2**: AlphaFold structure of RdRp with motifs A (blue), B (green), and C (red) visualized. Most of the residues of this AlphaFold structure had a pLDDT > 90. 
+
+
+The novel dicistrovirus has an Rdrp sOTU number of u257616 and SRA number SRR13764152. From the blast runs with just the palmprint sequence, it hit the Ancient Northwest Territories cripavirus species and the Dicistroviridae family.
+
+I ran virusRdRpCandidates.fa, then I took Node_1143 which had all three palmprint motifs A, B, and C and the longest sequence. I used the ammino acid sequence of Node_1143 for this problem. I determined all the other RdRP sequences in the dicistroviridae family using NCBI (I put the NODE_1143 sequence into BLASTp then I clicked taxonomy and )
+
+I put its sequence with the sequence of the hit into MUSCLE. Then I visualized the alignment in Jalview.
+
+![](Images/Sequence_alignment_final.png)
```
Comment (ababaian) [inline virTemplate.md]: This MSA is difficult to see in a markdown as it's very wide.

**Comment by MAM122** [comment]
Thank you Professor! I edited my document. As for the image of the alignment.png, I forgot to delete that image (the final sequence alignment is "sequence_alignment_final.png").

**Comment by MAM122** [comment]
In my gitlab account, I also just deleted the alignment.png file to avoid confusion. I hope that is okay.

**Review by ababaian — APPROVED** [review]
Nice :)

**Pull request merged**